### ISSUE TITLE
Setting environment variable from a job

### ISSUE THREAD

**Issue by waghanza** [tbrand/neph#70]
Hi @notramo,

I need to execute a job, and to set `output` as `environment` variable.

I think the `README` is a bit outdated, https://github.com/tbrand/neph/wiki/Set-a-job-result-to-env-vars

I can help with the wiki, since I use `neph`.

Do you plan to push specs ?

Regards,

**Comment by notramo** [comment]
@waghanza It is not implemented yet. (The wiki is outdated, it was written for the old version.)
Do you want to implement it?

**Comment by waghanza** [comment]
@notramo I'm not sure I can ;-) 
I'm still focus on `which_is_the_fastest`

**Comment by notramo** [comment]
@waghanza No problem, I will implement it.

I recently switched to Void Linux, and I like it very much. The problem is that it still ships Crystal 0.24.1 (https://github.com/void-linux/void-packages/issues/306). In Void everything works, and does its own job. Also, it's fast, simple, stable, and comfortable, so I don't want to switch back to Arch only because Crystal 0.25.

I can't work on the project by coding until Crystal 0.25 lands in the Void repo. Until this, I will improve the documentation, and design new features. E.g. I recently found a solution to implement the `simple` (a.k.a. `CI`) output format in a clean and simple way. It's almost ready for coding, only needs some modifications to be able to power the `full` (animated) mode also.
